### PR TITLE
Fix e2e tests broken in #17865

### PR DESCRIPTION
**Pull request by fgrzadkowski**
It was broken in #17865

@davidopp @HaiyangDING - FYI

### CODE REVIEW

File: test/e2e/scheduler_predicates.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func verifyResult(c *client.Client, podName string, ns string) {
 			"involvedObject.kind":      "Pod",
 			"involvedObject.name":      podName,
 			"involvedObject.namespace": ns,
-			"source":                   "scheduler",
+			"source":                   api.DefaultSchedulerName,
```
Comment (wojtek-t) [inline test/e2e/scheduler_predicates.go]: I think we shouldn't change this one, since this is about 1.0 branch, which didn't change.

Comment (fgrzadkowski) [inline test/e2e/scheduler_predicates.go]: Good catch!

**Comment by fgrzadkowski** [comment]
@wojtek-t Done. PTAL

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**